evennia/server/portal/webclient.py

```diff
@@ -82,9 +82,12 @@ def get_client_session(self):
             logger.log_trace(str(self))
             return None
 
-        self.csessid, *browserstr = webarg.split("&", 1)
-        if browserstr:
-            self.browserstr = str(browserstr[0])
+        self.csessid, *cargs = webarg.split("&", 2)
+        if len(cargs) == 1:
+            self.browserstr = str(cargs[0])
+        elif len(cargs) == 2:
+            self.cuid = str(cargs[0])
```

Comment:
Hm. We now hae cuid, cusessid, csessid, browserstr,  ... it's starting to be confusing to track what's what here. While csessid is probably something we need to keep for backwards compatibility, maybe it would be worthwhile to start renaming local variables to something more useful, like `client_session_id`, `webclient_session_id` etc?

Reply:
Sure! I'd be happy to rename them. `csessid` is tracking the browser's login session and needs to be kept separate for authentication, but I can alias. `cuid` is the individual client page's ID and I can name absolutely anything. What would you prefer for each?

Reply:
@InspectorCaracal I don't think it would hurt to be explicit here; so something like `webclient_page_id` instead of `cuid` and so on - something that actually tells you what the id points to.